internal/provider/kubernetes/filters.go

```diff
@@ -57,7 +57,7 @@ func (r *gatewayAPIReconciler) getExtensionBackendResources(ctx context.Context)
 		uExtResourceList := &unstructured.UnstructuredList{}
 		uExtResourceList.SetGroupVersionKind(gvk)
 		if err := r.client.List(ctx, uExtResourceList); err != nil {
-			r.log.Info("no associated backend resources found for %s", gvk.String())
```

Comment:
This is to fix DPANIC of improper key–value pairs.

Reply:
I'll address these in a separate PR https://github.com/envoyproxy/gateway/pull/6846